plexpy/notification_handler.py

```diff
@@ -1320,7 +1321,7 @@ def build_notify_text(subject='', body='', notify_action=None, parameters=None,
             logger.exception("Tautulli NotificationHandler :: Unable to parse custom script arguments: %s. Using fallback." % e)
             script_args = []
 
-    elif agent_id == 25:
+    elif agent_id == 25 or as_json:
```

Comment:
Logger messages below still say "webhook". Format the logger messages with the correct agent.

```py
agent = 'MQTT' if agent_id == 23 else 'webhook'
```
```py
logger.error("Tautulli NotificationHandler :: Unable to parse custom %s json header data: %s. Using fallback." % (agent, e))
```